pulsar-broker-common/src/main/java/org/apache/pulsar/common/stats/package-info.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ * Provides metric utility classes.
+ */
+package org.apache.pulsar.common.stats;
```

Comment:
since this is the `pulsar-broker-common` module, I think that we should use a package under `org.apache.pulsar.broker` to prevent the problem of split packages. 
`pulsar-common` already contains the `org.apache.pulsar.common.stats` package, so that package name is already taken. Split packages aren't allowed in Java Platform Modules System (JPMS, Java 9 modules) either.

Reply:
I don't understand the problem

Reply:
Couldn't we address this package split issue already in this PR before proceeding?

Reply:
Oh I see that no new split package anymore in this PR, problem resolved.